library/Haste/Ajax/ReloadHelper.php

```diff
@@ -0,0 +1,235 @@
+<?php
+
+namespace Haste\Ajax;
+
+use Haste\Http\Response\JsonResponse;
+
+class ReloadHelper
+{
+    const TYPE_CONTENT_ELEMENT = 'ce';
+    const TYPE_FRONTEND_MODULE = 'fmd';
```

Comment:
Actually, the types are totally unnecessary. The argument must simply be a unique ID, which means `ce_17` or similar. As far as I could see, we never need "all of one type"?